python/astropy.rst

```diff
@@ -28,6 +24,15 @@ The following packages can be used internally in packages if they do not leak in
 * :mod:`astropy.cosmology`
 * :mod:`astropy.visualization`
 
+For reading and writing files in FITS format, both :mod:`astropy.io.fits` (use of ``pyfits`` is now deprecated) and `fitsio`_ are allowed.
```

Comment:
I'm guessing at this point we can drop the ``pyfits`` comment since it's been dead for years and many people don't even know it existed.

Reply:
Am I old enough to have actually used it? 😢